### DISCUSSION TITLE
Second plot not starting

### DISCUSSION THREAD

**Discussion by UnPure** [swar/Swar-Chia-Plot-Manager#156] [Q&A]
```
    stagger_minutes: 60
    max_for_phase_1: 2
```
The first job started without any problems and is running for a little over 80 minutes now but even though the view keeps refreshing no second job is starting. Do the 60 minutes start right at the beginning of the first job or at a different point? How can I see whats causing the problem?

//edit: After 1:31:59 it started a second job. I'll keep an eye on it and see if all other jobs start as soon as they should.
//edit 2: I just noticed that the ~92 minutes that it took are the exact amount of time the first job took to get out of stage 1. That means that for some reason it is ignoring `max_for_phase_1: 2` and keeping it at 1. The second job is running for over 1 hour now already and so far there is no third job started, I'm expecting the 3rd job to start as soon as job 2 is done with stage 1.

**Comment by raid5** [comment]
Can you include the entire `config.yaml` file?

**Reply by UnPure** [reply]
https://pastebin.com/v50hcgvb

**Reply by raid5** [reply]
hmm yea that all looks correct to me, so maybe someone else will chime in.

I'm not sure if Plot Manager has its own set of logs you can view or not. That would be a good feature though.

**Reply by selop** [reply]
Not sure if this combination is causing trouble:

```
max_concurrent: 7
max_concurrent_with_start_early: 8
```

Have you tried `max_concurrent: 8` as well?

You could try to lower the manager log level from ERROR to INFO and check the `manager.log` which is in the same directory as the `config.yaml`.

**Reply by UnPure** [reply]
Why would this combination cause trouble? I'm using it the way it is supposed. I don't want more than 7 at any given time unless it is starting the 8th phase early. Even the default config has max_concurrent + 1, thats the whole purpose of it.

Over night my manager stopped on its own for some reason, I can't find why, no errors and no obvious problems. The view was open and still running but no active jobs running. I just started the manager again and I'll keep an eye on it to see if it is doing the same again today. To "fix" the problem temporarely yesterday I changed the max phase 1 entry to 3 and reloaded the config which allowed it to start 2 .
